src/training_raster_clipper/implementation/iatraoui.py

```diff
@@ -38,8 +42,24 @@ def load_sentinel_data(
         xr.DataArray: A DataArray containing the 3 RGB bands from the visible spectrum
     """
 
-    ...  # TODO
-    raise NotImplementedError
+    # ...  # TODO
+    # raise NotImplementedError
+    # p = Path(sentinel_product_location)
+    # paths_list = [x for x in p.iterdir() if x.is_file()]
+    # colors_list = [str(x).split('_')[-2] for x in paths_list]
+    # data_dict = dict(zip(colors_list, paths_list))
+    # data_arrays_list = 
+    paths_list = {
+        band_name : list(sentinel_product_location.glob(f'GRANULE/*/IMG_DATA/R{resolution}m/*_{band_name}_*'))[0]
+        for band_name in band_names
+    }
+    data_arrays_list = [open_rasterio(paths_list[band_name]).assign_coords({'band' : [band_name]}) for band_name in band_names]
+    rasters_data_array = xr.concat(data_arrays_list, dim = 'band')
+    rasters_data_array = rasters_data_array.where(rasters_data_array != 0, np.float32(np.nan))
+    RADIO_ADD_OFFSET = -1000
```

Comment:
Variable names in uppercase should be reserved for top-level variables.
You can either move them on top of the file, or lowercase them